docs/ml-frequent-pattern-mining.md

```diff
@@ -85,3 +85,56 @@ Refer to the [R API docs](api/R/spark.fpGrowth.html) for more details.
 </div>
 
 </div>
+
+## PrefixSpan
```

Comment:
is this not in the doc for branch-2.4? we might need a quick fix on the branch then before the next RC, I think